templates/10-canal.conflist

```diff
@@ -5,6 +5,7 @@
       "type": "flannel",
       "delegate": {
         "type": "calico",
+        "hairpinMode": true,
```

Comment:
Does this work? I'm under the impression that `hairpinMode` is a config key for the `bridge` plugin, but in this case flannel is delegating to the `calico` plugin instead.

Reply:
I am going to give it another try but in my tests this change and flannel restart worked.

Reply:
You are right that this is not needed. In the canal bundle the deployment is initially failing with 502s. If you delete the monitoring-influxdb-grafana-v4 pod k8s recreates the pod and it works as expected afterwards. Closing this PR. Although the symptoms are the same the hairpin mode is not the cure...